packages/core/src/components/button/buttonGroup.tsx

```diff
@@ -41,13 +49,15 @@ export class ButtonGroup extends React.PureComponent<IButtonGroupProps, {}> {
     public static displayName = "Blueprint2.ButtonGroup";
 
     public render() {
-        const { className, fill, minimal, large, vertical, ...htmlProps } = this.props;
+        const { align, className, fill, minimal, large, vertical, ...htmlProps } = this.props;
         const buttonGroupClasses = classNames(
             Classes.BUTTON_GROUP,
             {
+                [Classes.ALIGN_LEFT]: align === "left",
+                [Classes.ALIGN_RIGHT]: align === "right",
                 [Classes.FILL]: fill,
-                [Classes.MINIMAL]: minimal,
                 [Classes.LARGE]: large,
+                [Classes.MINIMAL]: minimal,
```

Comment:
small?

Reply:
would prefer not to add tangential new props in this PR.
i plan to add modifier props across the board after 2.0 (non-breaking)